Marlin/G26_Mesh_Validation_Tool.cpp

```diff
@@ -555,10 +546,16 @@
     return G26_OK;
   }
 
+  float valid_trig_angle(float d) {
+    while (d > 360.0) d -= 360.0;
+    while (d < 0.0) d += 360.0;
+    return d;
+  }
+
   /**
    * G26: Mesh Validation Pattern generation.
    *
-   * Used to interactively edit UBL's Mesh by placing the
+   * Used to interactively edit the mesh by placing the
    * nozzle in a problem area and doing a G29 P4 R command.
```

Comment:
@Roxy-3D — I guess `G26` doesn't do `G29 P4 R` when UBL is disabled. What should this comment say instead?